WooCommerce/Classes/System/WooTabContainerController.swift

```diff
@@ -0,0 +1,35 @@
+import UIKit
+
+/// Container for a Woo tab, shown as the root view controller of one of the tabs.
+/// Provided as an alternative to `WooTabNavigationController`, for root controllers which should not be in a nav view
+/// For example, a Split View, which will not work correctly on iPhone when wrapped in a navigation view.
+/// This wraps a controller which can be replaced when the selected site changes.
+class WooTabContainerController: UIViewController {
```

Comment:
```suggestion
final class TabContainerController: UIViewController {
```

Do we want to allow subclassing for this one? Otherwise we can mark it as final. 

Question: Do you think we should remove the `Woo` part from the class name? Since is not really a wrapper for a WooTab  or other concrete implementations, but for any `UIViewController`. On the contrary, `WooTabNavigationController` inherits from `WooNavigationController` so the name goes along.

Reply:
Yeah, that makes sense since the WooNavigationController is so named because it adds Woo styling (among other things.)